app/views/notifier/notify.html.haml

```diff
@@ -18,6 +18,7 @@
   %br/
   = link_to "View this message in your inbox", notification_url(@notification)
   %br/
-  = link_to "Turn off these notifications", edit_member_registration_url
+  = link_to "Turn off these notifications", unsubscribe_member_url(@signed_message)
```

Comment:
Or even better, "Unsubscribe from direct message notifications" for this one, "... from planting reminders" for planting reminders, etc.  So it's clear you're unsubscribing from one particular kind of message.